docs/07_maintainers_view.md

```diff
@@ -1,78 +1,70 @@
+*Disclaimer: This is a meme. Don't take it personally.*
 
- __Always go for Bugfixes__, as the best application with the best features
+ __Always prioritize fixing bugs__, as the best application with the best features
    does not help much if it is broken, or annoying to use. Now if a program
-   is in an early stage it is quite understandable that many things brake. This
-   is one reason why NewPipe still has no 1 in the beginning of its version
+   is in an early stage it is quite understandable that many things break. This
+   is one reason why NewPipe still has no "1" in the beginning of its version
    number.
-   However by now NewPipe is in a stage where there should be a strong focus on
+   By now, NewPipe is in a stage where there should be a strong focus on
    stability.
 
-1. If there are multiple Pull requests open, check the ones with the bugfixes first.
-2. Do not add to much features every version, as every feature will inevitable
-    introduce more bugs. It is quite ok, if PRs stay open for a while (not to long though).
+1. If there are multiple Pull Requests open, check the ones with bugfixes first.
+2. Do not add too many features every version, as every feature will inevitably
+    introduce more bugs. It is OK if PRs stay open for a while, but don't leave them open for too long.
 3. If there are bugs that are stale, or open for a while bump them from time
    to time, so devs know that there is still something left to fix.
-4. Never accept bugs. From my perception the community does not like to fix bugs, this is why you as a maintainer should
-   especially focus on perusing bugs. 
+4. Never merge PRs with known issues. From my perception the community does not like to fix bugs, this is why you as a maintainer should
+   especially focus on pursuing bugs.
 
 
 ### Features
 
-Well features are also something that can cause a headache. You should always see adding features critical and question
-whether that features does make sense, is useful and would actually be an advantage for the app. You should not blindly
-say yes to features even if they are small, however you should also not directly say no as well. Think about it, may
-be even for days before deciding whether you want to accept a feature or not. If you are not sure, try it, look into the
-code, speak with the developer, and then make a decision and justify it. The criteria whether to add a feature or not
-should be:
+Features are also something that can cause a headache. You should not blindly
+say yes to features, even if they are small, but you should also not immediately say no. If you are not sure, try the feature, look into the
+code, speak with the developer, and then make a decision. When considering a feature, ask yourself the following questions:
+
+- Was the feature requested by only a few, or by many?
+	- Avoid introducing niche features to satisfy a small handful of users.
```

Comment:
```suggestion
	- Avoid introducing nice features to satisfy a small handful of users.
```

Reply:
Oh it's late. I should stop reviewing :D

Reply:
That word is correct as is.

niche - denoting or relating to products, services, or interests that appeal to a small, specialized section of the population.

Reply:
He should know. The word exists in German as well. There it's written "Nische".

Reply:
Yes, noticed it too. That's why I decided to stop for today 😆